cohort-engine/src/main/java/com/ibm/cohort/engine/measure/MeasureEvaluator.java

```diff
@@ -72,14 +74,16 @@ public MeasureReport evaluatePatientMeasure(Measure measure, String patientId, S
 		LibraryLoader libraryLoader = LibraryHelper.createLibraryLoader(libraryResolutionProvider);
 
 		EvaluationProviderFactory factory = new ProviderFactory(dataClient, terminologyClient);
-		MeasureEvaluationSeed seed = new MeasureEvaluationSeed(factory, libraryLoader, libraryResolutionProvider);
+		MeasureEvaluationSeeder seeder = new MeasureEvaluationSeeder(factory, libraryLoader, libraryResolutionProvider);
+		seeder.disableDebugLogging();
 
 		// TODO - consider talking with OSS project about making source, user, and pass
 		// a properties collection for more versatile configuration of the underlying
 		// providers. For example, we need an additional custom HTTP header for
 		// configuration of our FHIR server.
-		seed.setup(measure, periodStart, periodEnd, /* productLine= */"ProductLine", /* source= */"", /* user= */"",
-				/* pass= */"");
+		IMeasureEvaluationSeed seed = seeder.create(measure, periodStart, periodEnd, "ProductLine");
+//		seed.setup(measure, periodStart, periodEnd, /* productLine= */"ProductLine", /* source= */"", /* user= */"",
```

Comment:
Can we get rid of this commented out code or add another comment about why it might be used in the future?